nixos/modules/services/databases/postgresql.md

```diff
@@ -354,6 +354,59 @@ self: super: {
 }
 ```
 
+You can add a custom postgresql extension to an environment by calling `postgresqlPackages.callPackage` on the derivation.
+In addition to the correct postgresql package, `callPackage` will provide additional functions to build those extensions:
+
+- `postgresqlBuildExtension`, extending `mkDerivation` for C and sql extensions
+- `buildPgrxExtension`, extending `mkDerivation`q for pgrx (rust) extensions
+- `postgresqlTestExtension`, an helper to test these extensions on a postgresql instance.
```

Comment:
Technically, `buildPgrxExtension` is available in the global scope. Although.. we might want to change that, I'd think.